utils/nm-dispatcher.in

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/sh
+# This is a NetworkManager dispatcher script for the mstpd daemon
+# to apply any configuration to bridges based on the values in
+# /etc/sysconfig/network-scripts/bridge-stp
+
+export LC_ALL=C
+
+interface=$1
+action=$2
+
+config_cmd='@configbridgefile@'
+
+if [ "$action" = "up" -a -n "$interface" ]; then
```

Comment:
Should use `[ ... ] && [ ... ]` instead of `[ ... -a ... ]`.  I don't remember the exact reasoning for this - POSIX compliance or something - but the rest of the mstpd scripts do it that way, so let's do that for consistency if nothing else.

Reply:
Fixed... consistency is good :)